### PR TITLE
Update: add and remove tutorials

### PR DESCRIPTION
**Pull request by estelle**
Minor changes. I added a few inline github comments to explain why I removed stuff
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/tutorials/index.md [hunk 1]
```diff
@@ -103,20 +112,10 @@ These resources are created by forward-thinking companies and web developers who
   - : Apply rotation, skewing, scaling, and translation using CSS.
 - [CSS transitions](/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions)
   - : CSS transitions provide a way to animate changes to CSS properties, instead of having the changes take effect instantly.
-
-<!---->
-
-- [Quick Guide to Implement Web Fonts  (with @font-face)](https://www.html5rocks.com/en/tutorials/webfonts/quick/)
```
Comment (estelle) [inline files/en-us/web/tutorials/index.md:109]: redirects to web.dev

File: files/en-us/web/tutorials/index.md [hunk 2]
```diff
@@ -103,20 +112,10 @@ These resources are created by forward-thinking companies and web developers who
...
-  - : The CSS @font-face feature allows you to use custom typefaces on the web in an accessible, manipulatable, and scalable way.
-- [Starting to Write CSS](https://davidwalsh.name/starting-css)
-  - : An introduction to tools and methodologies to write more succinct, maintainable, and scalable CSS.
```
Comment (estelle) [inline files/en-us/web/tutorials/index.md:112]: too old

File: files/en-us/web/tutorials/index.md [hunk 3]
```diff
@@ -103,20 +112,10 @@ These resources are created by forward-thinking companies and web developers who
...
-
-<!---->
-
 - [Canvas tutorial](/en-US/docs/Web/API/Canvas_API/Tutorial)
   - : Learn how to draw graphics using scripting using the canvas element.
-- [HTML5 Doctor](https://html5doctor.com/)
-  - : Articles about using HTML right now.
```
Comment (estelle) [inline files/en-us/web/tutorials/index.md:119]: hasn't been updated in 6 years

File: files/en-us/web/tutorials/index.md [hunk 4]
```diff
@@ -103,20 +112,10 @@ These resources are created by forward-thinking companies and web developers who
...
+- [Cascade layers](/en-US/docs/Learn/CSS/Building_blocks/Cascade_layers)
+  - How to control the cascade, specificity, and inheritance within developer styles using cascade layers.
```
Comment (estelle) [inline files/en-us/web/tutorials/index.md:118]: this is currently a dead link, but the tutorial will be done in the next 4 weeks

Comment (teoli2003) [reply files/en-us/web/tutorials/index.md:118]: Can you add it in the PR that will add the tutorial? That way we don't have an extra red link?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just the future tutorial. (The other are fine to remove)

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**